### DISCUSSION TITLE
elastalert-container WARNING:elasticsearch:DELETE https://[blah]/_search/scroll [status:404 request:0.016s]

### DISCUSSION THREAD

**Discussion by markwheal** [jertel/elastalert2#1268] [Q&A] [answered]
I have two instances of Elastalert2 running, both querying the same Elasticsearch instance.

I am seeing this error from one of the two Elastalert2 instances, but never from the other instance. Both instances are reading and writing to mutually exclusive/unique index names (the Elastalert indexes have prefixes to make each one unique).

Is this something to be concerned about?

Thanks
Mark

**Comment by jertel** [comment, marked as answer]
Possibly. It sounds like Elasticsearch may be reusing scroll IDs - perhaps it generates the scroll ID by hashing the request query, and since both ElastAlert 2 instances are making the same query, they both get the same scroll ID. The risk here is that the two instances are manipulating the same scroll, potentially resulting in incomplete search results being returned.

Are you using two ElastAlert 2 instances for redundancy? If so, did you consider replicating the Elastic data to a second cluster and having the second ElastAlert 2 instance query that second cluster instead? If you have a massive data store then it may not be feasible. However, if you aren't already using replica shards anyway, this could be a benefit in that you gain redundancy in the datastore as well as fixing the scroll issue.
